### PR TITLE
Recommend CRs update non-code documentation.

### PR DESCRIPTION
**Pull request by greglo**

### CODE REVIEW

**Comment by greglo** [comment]
@zoubeiri

**Comment by zoubeiri** [comment]
@eparish for CR

File: code-reviews/readme.md [hunk 1]
```diff
@@ -267,6 +267,12 @@ consider in a code review:
   (Think of a time you saw an excellent, or truly terrible, commit
   message or comment.)
 
+- **Was the external documentation updated?**
+  If your project maintains a readme, changelog, or other documentation,
```
Comment (eparish) [inline code-reviews/readme.md]: Is readme usually styled as all caps?

**Comment by eparish** [comment]
One minor question, otherwise looks good!

**Pull request merged**